WooCommerce/Classes/ViewModels/ManualTrackingViewModel.swift

```diff
@@ -63,7 +63,8 @@ protocol ManualTrackingViewModel {
 
     var shipmentProvider: ShipmentTrackingProvider? { get set }
     var shipmentProviderGroupName: String? { get set }
-
+    var isEmptyState: Bool { get }
```

Comment:
Is this property `isEmptyState` necessary?

Reply:
I guess I could get rid of it by using the "not operator" on `canCommit`, right? As is basically the opposite?